invenio_deposit/jsonschemas/deposits/deposit-v1.0.0.json

```diff
@@ -11,19 +11,6 @@
         "id": {
           "type": "string",
           "name": "id"
-        },
```

Comment:
Please remove this change. We are going to update deposit schema in other PR. Thanks